### PR TITLE
AWS how-to-guides EC2

### PR DESCRIPTION
**Pull request by garvit3835**

### CODE REVIEW

**Comment by vikasgoswami2** [comment]
@mishushakov Please review this and let me know the feedback.

File: how-to-guides/cloud-services/aws/connect-to-an-ec2-instance.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+## New EC2 Machine
+
+### Step 1: Creating an EC2 Instance
+
+1. Follow the [Connecting to AWS](../../existing-network/connecting-to-aws.md) guide.
+2. Go to **Amazon EC2 > Instances > Launch Instances**.
+3. Enter name of the instance.
+4. Use one of the [supported linux distros](https://web.archive.org/web/20230927004045/https://tailscale.com/kb/1017/install).
+5. Choose **Instance type**.
```
Comment (mishushakov) [inline how-to-guides/cloud-services/aws/connect-to-an-ec2-instance.md:34]: the Instance type

File: how-to-guides/cloud-services/aws/connect-to-an-ec2-instance.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+6. Create a new **key pair** and save it.
+7. In **Network settings**, click on **edit** and choose the VPC where your bastion host is running, the private subnet and the security group which allows inbound access to VPC.
```
Comment (mishushakov) [inline how-to-guides/cloud-services/aws/connect-to-an-ec2-instance.md:36]: the Network settings, the VPC

File: how-to-guides/cloud-services/aws/connect-to-an-ec2-instance.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+### Step 2: Accessing EC2 from a DevBox
+1. Go to **DevBox** and make a copy of the key pair.
```
Comment (mishushakov) [inline how-to-guides/cloud-services/aws/connect-to-an-ec2-instance.md:39]: the DevBox

File: how-to-guides/cloud-services/aws/connect-to-an-ec2-instance.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+2. Connect to this EC2 using ssh and the new key pair:\
```
Comment (mishushakov) [inline how-to-guides/cloud-services/aws/connect-to-an-ec2-instance.md:40]: Connect to the desired EC2 instance

**Review by mishushakov — CHANGES_REQUESTED** [review]
looks good to me, only minor grammar fixes

**Comment by vikasgoswami2** [comment]
@mishushakov please check now. I made the changes suggested by you.

**Review by mishushakov — APPROVED** [review]

**Comment by mishushakov** [comment]
Perfect, thank you!

**Pull request merged**